Shared/Samples/Measure distance in scene/MeasureDistanceInSceneView.swift

```diff
@@ -0,0 +1,175 @@
+                            verticalDistanceText = measurements.verticalDistance.formatted(.distance)
+                        }
+                    }
+                    .overlay(alignment: .top) {
+                        // Instruction text.
+                        Text(instructionText)
+                            .frame(maxWidth: .infinity, alignment: .center)
+                            .padding(8)
+                            .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+                    }
+            }
+            
+            // Distance texts.
+            Text("Direct: \(directDistanceText)")
+            Text("Horizontal: \(horizontalDistanceText)")
+            Text("Vertical: \(verticalDistanceText)")
+            
+            // Unit system picker.
+            Picker("", selection: $unitSystemSelection) {
+                Text("Imperial").tag(UnitSystem.imperial)
+                Text("Metric").tag(UnitSystem.metric)
+            }
+            .pickerStyle(.segmented)
+            .padding()
+            .onChange(of: unitSystemSelection) { _ in
+                model.locationDistanceMeasurement.unitSystem = unitSystemSelection
+            }
+        }
+    }
+}
+
+private extension MeasureDistanceInSceneView {
+    /// The view model for the sample.
+    class Model: ObservableObject {
+        /// A scene with an imagery basemap.
+        let scene = {
+            let scene = ArcGIS.Scene(basemapStyle: .arcGISTopographic)
+            
+            // Add elevation source to the base surface of the scene with the service URL.
+            let elevationSource = ArcGISTiledElevationSource(url: .worldElevationService)
+            scene.baseSurface.addElevationSource(elevationSource)
+            
+            // Create the building layer and add it to the scene.
+            let buildingsLayer = ArcGISSceneLayer(url: .brestBuildingsService)
+            scene.addOperationalLayer(buildingsLayer)
+            
+            return scene
+        }()
+        
+        /// An analysis overlay for location distance measurement.
+        let analysisOverlay = AnalysisOverlay()
+        /// The location distance measurement.
+        let locationDistanceMeasurement = LocationDistanceMeasurement(
+            startLocation: Point(x: -4.494677, y: 48.384472, z: 24.772694, spatialReference: .wgs84),
+            endLocation: Point(x: -4.495646, y: 48.384377, z: 58.501115, spatialReference: .wgs84)
+        )
+        
+        init() {
+            // Set scene to the viewpoint specified by the location distance measurement.
+            let lookAtPoint = Envelope(
+                min: locationDistanceMeasurement.startLocation,
+                max: locationDistanceMeasurement.endLocation
+            ).center
+            let camera = Camera(lookingAt: lookAtPoint, distance: 200, heading: 0, pitch: 45, roll: 0)
+            scene.initialViewpoint = Viewpoint(boundingGeometry: lookAtPoint, camera: camera)
```

Comment:
```suggestion
            scene.initialViewpoint = Viewpoint(latitude: .nan, longitude: .nan, scale: .nan, camera: camera)
```
I don't like a point being the bounding geometry of a map or scene view as "bounding" sounds like an area or polygon to me. I like to explicitly set the unrelated parameter to `nan` to emphasize that a scene viewpoint can be solely determined by the camera, but it is up to you.